flatland_plugins/include/flatland_plugins/laser.h

```diff
@@ -79,6 +79,7 @@ class Laser : public ModelPlugin, public b2RayCastCallback {
   std::string frame_id_;  ///< laser frame id name
   bool broadcast_tf_;     ///< whether to broadcast laser origin w.r.t body
   uint16_t layers_bits_;  ///< for setting the layers where laser will function
+  uint16_t reflectance_layers_bits_;  ///< for setting reflectance layers. if laser hits those layers, intensity will be high
```

Comment:
It may be better to rename all of the "reflectance*" variables "intensity*" since although the intensity return is related to reflectance, the value output is still intensity.

Reply:
Depends on the naming convention you want to maintain.

I think it is better this way, because it is closer to reality, but for newcomers maybe simpler to maintain the same name for the variable and the layer.